src/test/java/org/shikshalokam/backend/ep/ElevateProjectBaseTest.java

```diff
@@ -19,6 +22,8 @@ public class ElevateProjectBaseTest extends MentorBase {
     public static String User_ID = null;
     public static String BASE_URL = PropertyLoader.PROP_LIST.getProperty("elevate.qa.api.base.url");
     public static String INTERNAL_ACCESS_TOKEN = PropertyLoader.PROP_LIST.getProperty("elevate.internalaccesstoken");
+    public static String CRUDEntityType_Name = PropertyLoader.PROP_LIST.getProperty("elevate.qa.CRUDEntitytype.name");
```

Comment:
why you need here ?
it should be at Testcase level

Reply:
actually I will be calling it while verifying the bulk entity type creation hence I put it in base class